frontend/src/lib/lemon-ui/LemonCollapse/index.ts

```diff
@@ -0,0 +1 @@
+export * from './LemonCollapse'
```

Comment:
As far as I know we want to avoid `index.ts` files and instead import from the "named" file.

Would be good to add the LemonCollaps to `frontend/@posthog/lemon-ui/src/index.ts` as well :)

Reply:
Hmm, to me it seems that the ergonomics of importing from `lib/lemon-ui/LemonCollapse` are better than that of `lib/lemon-ui/LemonCollapse/LemonCollapse`, and this takes just a one-line `index.ts` – so I'd actually propose always using `index.ts`.

Good point with `@posthog/lemon-ui`, added!